opentdf-dev.yaml

```diff
@@ -130,6 +130,7 @@ server:
       - Content-Type
       - X-CSRF-Token
       - X-Request-ID
+      - Connect-Protocol-Version
```

Comment:
Do we need to add this header to the rest of the config examples?

Reply:
yeah, it's a good idea to keep things consistent